web/forms.py

```
@@ -832,38 +833,96 @@ def clean_topics(self):
-    email = forms.EmailField(
-        widget=TailwindEmailInput(
+    course_image = forms.ImageField(
+        required=True,
+        validators=[FileExtensionValidator(["jpg", "jpeg", "png", "gif"])],
+        widget=TailwindFileInput(
             attrs={
-                "placeholder": "Your email address",
+                "accept": "image/*",
+                "help_text": "Upload a course image (required)",
                 "class": "block w-full border rounded p-2 focus:outline-none focus:ring-2 focus:ring-indigo-500",
             }
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)